tests/random_temperature_perturbation.prm

```diff
@@ -0,0 +1,44 @@
+# Test to check the status of the random initial temperature perturbation
+# plugin.
+
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = left, right, bottom, top
+end
+
+subsection Mesh refinement
+  set Initial global refinement = 3
+end
+
+subsection Postprocess
+  set List of postprocessors = temperature statistics
+end
+
+subsection Material model
+  set Model name = simpler
+end
+
+set End time = 0
+subsection Geometry model
+  set Model name = box
+end
+
+subsection Gravity model
+  set Model name = vertical
+end
+
+subsection Initial temperature model
+  set List of model names = function, random perturbation
+  subsection Random perturbation
+    set Magnitude = 0.2
+  end
```

Comment:
Do you need to say how they are composed (or at least could we do so explicitly)?